charts/fleet/templates/rbac_gitjob.yaml

```diff
@@ -43,6 +43,21 @@ rules:
       - 'events'
     verbs:
       - '*'
+  - apiGroups:
+      - ""
+    resources:
+      - serviceaccounts
+    verbs:
+      - "create"
+  - apiGroups:
+      - rbac.authorization.k8s.io
+    resources:
+      - clusterroles
+      - clusterrolebindings
```

Comment:
Are these really being used?

Reply:
you're right... those 2 are not needed... ( :star: the wonders of copy/paste :star2: ). Thanks for catching it!